ext/nokogiri/xml_io.c

```diff
@@ -2,12 +2,13 @@
 
 static ID id_read, id_write;
 
-VALUE read_check(VALUE *args) {
+VALUE read_check(VALUE _args) {
```

Comment:
There's a convention where arguments that are known not to be used begin with an underscore. In TypeScript, this disables compiler warnings (or errors based on compiler flags) for those unused function arguments (not sure if this is the case with C compilers). In Ruby, if you use Rubocop, similar behaviour is observed.

Reply:
There's no such convention in C, but I changed it nonetheless to avoid any misinterpretation.